backend/tests/integration/records/api/test_api_authors.py

```diff
@@ -191,3 +190,14 @@ def test_get_author_papers(inspire_app):
     author_papers = author.get_papers_uuids()
     assert str(lit_1.id) in author_papers
     assert str(lit_2.id) not in author_papers
+
+
+def test_dump_for_es(inspire_app):
+    additional_fields = {
+        "positions": [{"institution": "CERN", "current": True}, {"institution": "DESY"}]
+    }
+    data = faker.record("aut", data=additional_fields)
+    record = AuthorsRecord.create(data)
+    dump = record.serialize_for_es()
+
+    assert "CERN" in dump["author_suggest"]["input"]
```

Comment:
do we really want that? what goes into `input` used for `autocomplete`, so we want to autocomplete authors when user types an institutions. (I am not sure if they will, unless we tell them)

Reply:
if current `position` is just for display putting into `_source` and displaying on the UI is enough.

Reply:
I agree, but that was a part of refinement. I think it is not useful, as we have to do it in UI